tests/integration2/tunnel/tunnel_test.go

```diff
@@ -0,0 +1,351 @@
+package tunnel
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"testing"
+	"time"
+
+	"istio.io/istio/pkg/log"
+	apps2 "istio.io/istio/pkg/test/framework/runtime/components/apps"
+
+	v1 "k8s.io/api/core/v1"
+
+	"istio.io/istio/pilot/pkg/model"
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/api/components"
+	"istio.io/istio/pkg/test/framework/api/descriptors"
+	"istio.io/istio/pkg/test/framework/api/ids"
+	"istio.io/istio/pkg/test/framework/api/lifecycle"
+	"istio.io/istio/pkg/test/framework/runtime/components/environment/kube"
+	"istio.io/istio/pkg/test/framework/tmpl"
+)
+
+const (
+	clientSideEgressConfig = `
```

Comment:
IIUC, the configuration we have here is:

a->egress->ingress->b

Would it be possible to simplify this to just, provided we have a service configured for the ingress? i.e.:

a->ingress->b

Reply:
Our use case connects two clusters. Therefore, it's required to have an egress and an ingress involved.